src/test/cli/ceph-conf/help.t

```diff
@@ -26,7 +26,7 @@
   
   EXAMPLES
   [$] ceph-conf --name mon.0 -c /etc/ceph/ceph.conf 'mon addr' (re)
-  Find out what the value of 'mon add' is for monitor 0.
+  Find out what the value of 'mon addr' is for monitor 0.
```

Comment:
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes

also, please fold your commits into a single one.